arteria/models/runfolder_utils.py

```diff
@@ -116,6 +125,17 @@ def metadata(self):
 
         return metadata
 
+    def __setitem__(self, key, value):
+        return setattr(self, key, value)
+
+    def to_dict(self):
+        return {
+            "metadata": self.metadata,
+            "path": self.path,
+            "service_version": __version__,
```

Comment:
`service_version` is only related to the view we give to the user, so it should be added to the response by the handlers (in `serialize_runfolder_path`)